reladomographql/src/test/resources/test-schema.graphqls

```diff
@@ -0,0 +1,541 @@
+    processingDateFrom: TimestampAttribute
+    processingDateTo: TimestampAttribute
+    businessDate: TimestampAttribute
+    processingDate: TimestampAttribute
+    account: TamsMithraAccountFinder
+    and: TestBalanceNoAcmapFinder
+    or: TestBalanceNoAcmapFinder
+    tupleIn: [TestBalanceNoAcmapFinderTuple]
+}
+
+input TestBalanceNoAcmapFinderTuple
+{
+    accountId: String
+    productId: Int
+    positionType: Int
+    quantity: Float
+    businessDateFrom: Timestamp
+    businessDateTo: Timestamp
+    processingDateFrom: Timestamp
+    processingDateTo: Timestamp
+}
+
+#################### com.gs.fw.common.mithra.test.domain.Order ##################
+type Order
+{
+    orderId: Int
+    orderDate: Timestamp
+    userId: Int
+    description: String
+    state: String
+    trackingId: String
+    items: [OrderItem]
+    itemsForUserFive: [OrderItem]
+    productsWithoutInProgress: [Product]
+    itemsFilterByDate: [OrderItem]
+    children: [Order]
+    itemsforExistingStatus: [OrderItem]
+    itemsOrTest: [OrderItem]
+    itemsInTest: [OrderItem]
+    parent: Order
+}
+
+type OrderAggregate
+{
+    orderId: Int
+    varianceSampleOfOrderId: Int
+    maxOfOrderId: Int
+    avgOfOrderId: Int
+    minOfOrderId: Int
+    standardDeviationPopulationOfOrderId: Int
+    sumOfOrderId: Int
+    variancePopulationOfOrderId: Int
+    standardDeviationSampleOfOrderId: Int
+    orderDate: Timestamp
+    userId: Int
+    varianceSampleOfUserId: Int
+    maxOfUserId: Int
+    avgOfUserId: Int
+    minOfUserId: Int
+    standardDeviationPopulationOfUserId: Int
+    sumOfUserId: Int
+    variancePopulationOfUserId: Int
+    standardDeviationSampleOfUserId: Int
+    description: String
+    state: String
+    trackingId: String
+    items: [OrderItem]
+    itemsForUserFive: [OrderItem]
+    productsWithoutInProgress: [Product]
+    itemsFilterByDate: [OrderItem]
+    children: [Order]
+    itemsforExistingStatus: [OrderItem]
+    itemsOrTest: [OrderItem]
+    itemsInTest: [OrderItem]
+    parent: Order
+}
+
+input OrderInput
+{
+    orderId: Int
+    orderDate: Timestamp
+    userId: Int
+    description: String
+    state: String
+    trackingId: String
+}
+
+input OrderOrderBy
+{
+    orderId: AcsDesc
+    orderDate: AcsDesc
+    userId: AcsDesc
+    description: AcsDesc
+    state: AcsDesc
+    trackingId: AcsDesc
+}
+
+input OrderFinder
+{
+    orderId: IntegerAttribute
+    orderDate: TimestampAttribute
+    userId: IntegerAttribute
+    description: StringAttribute
+    state: StringAttribute
+    trackingId: StringAttribute
+    items: OrderItemFinder
+    itemsForUserFive: OrderItemFinder
+    productsWithoutInProgress: ProductFinder
+    itemsFilterByDate: OrderItemFinder
+    children: OrderFinder
+    itemsforExistingStatus: OrderItemFinder
+    itemsOrTest: OrderItemFinder
+    itemsInTest: OrderItemFinder
+    parent: OrderFinder
+    and: OrderFinder
+    or: OrderFinder
+    tupleIn: [OrderFinderTuple]
```

Comment:
Since `and` and `or` have the type of a single Finder, rather than an array of finders or exactly two finders, I can't figure out how you'd express order-of operations like (a || b) && (c || d).

Reply:
addressed to match the other implementations. 
https://github.com/vborisoff/reladomo/blob/graphql-real/reladomographql/docs/queries.md#filters are examples.